orbs/github-status.yml

```diff
@@ -19,7 +19,6 @@ commands:
           when: <<parameters.when>>
           name: "GitHub Status: <<parameters.state>>"
           command: |
-            set -euo pipefail
```

Comment:
is it breaking something? Wasn't the plan to actually call `set +u` here?

Reply:
`set +u` seems to be unnecessary, because it's never `-u`. Also neither are other flags, since it looks like cci runs bash with `-eo pipefall` flags by default.
nice explanation: https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425